docs/reference/glossary.rst

```diff
@@ -0,0 +1,97 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+.. contents::
+  :local:
+
+Flocker release types
+=====================
+
+.. note:: All Flocker documentation will refer the Production ready product, unless tagged Alpha or Beta.
+
+.. _alpha-definition:
+
+Alpha
+   This product or feature is a proof-of-concept; expect API instability; lack of security; product instability.
+
+.. _beta-definition:
+
+Beta
+   This product or feature is approaching readiness; expect minor API instability between versions; incomplete security; product stability for most common uses.
+
+.. _production-ready-definition:
+
+Production ready
+   This product or feature is suitable for production use; expect API stability between versions; robust security; product stability for all supported uses.
+
+Flocker terms
+=============
+
+.. _api-definition:
+
+Flocker API
+  A collection of RESTful API endpoints that allow you to configure a Flocker cluster programmatically with the tool and language of your choice.
+
+.. _cli-definition:
+
+Flocker CLI
+  Also known as the Flocker client.
```

Comment:
This is probably a good opportunity to get rid of the tiny number of places "Flocker client" is used.